test/topology_custom/test_tablets.py

```diff
@@ -23,3 +23,55 @@ async def test_tablet_change_replication_vnode_to_tablets(manager: ManagerClient
     with pytest.raises(InvalidRequest):
         await cql.run_async("ALTER KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1, 'initial_tablets': 1};")
 
+
+@pytest.mark.asyncio
+async def test_tablet_change_replication_strategy(manager: ManagerClient):
+    cfg = {'enable_user_defined_functions': False,
+           'experimental_features': ['tablets', 'consistent-topology-changes']}
+    server = await manager.server_add(config=cfg)
+
+    cql = manager.get_cql()
+    await cql.run_async("CREATE KEYSPACE test WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};")
```

Comment:
We'd like to kill SimpleStrategy ( https://github.com/scylladb/scylladb/issues/5224 , https://github.com/scylladb/scylladb/issues/8582 and others) - let's not add tests with it.

Reply:
(I don't remember which version we declared we'd do it - there's a guardrail to forbid creating it somewhere in the works?)

Reply:
Not necessarily to forbid, default action is to _warn_ about simple strategy (#15399 )

Reply:
Specifically the issue about not using SimpleStrategy in tests is https://github.com/scylladb/scylladb/issues/8638